dev/run_s3_archive.R

```diff
@@ -0,0 +1,39 @@
+logger::log_info("JOB START")
+print(
+  nflreadr::nflverse_sitrep(
+    c("nflversedata", "nflreadr", "piggyback", "minioclient")
+  )
+)
+logger::log_info("Downloading rds files from nflverse-data releases")
+purrr::walk(
+  piggyback::pb_releases("nflverse/nflverse-data")$tag_name,
```

Comment:
I'd prefer `nflversedata::gh_cli_release_tags()` for this and L19

Also since we call it twice, better save tag names to a variable?